test/extensions/secondorder/diag_ggn/diag_ggn_settings.py

```diff
@@ -9,8 +9,9 @@
 Shared settings are taken from `test.extensions.secondorder.secondorder_settings`.
 Additional local cases can be defined here through ``LOCAL_SETTINGS``.
 """
-from test.core.derivatives.utils import regression_targets
+from test.core.derivatives.utils import classification_targets, regression_targets
 from test.extensions.secondorder.secondorder_settings import SECONDORDER_SETTINGS
+from test.resnet.resnets_examples import ResNet1, ResNet2
```

Comment:
Rename `resnets_examples` into `resnet_cases` or `resnet_settings`. Otherwise it reads like the file contains a tutorial/demo.

Reply:
I renamed it to `resnet_cases`.